docker-compose.yml

```diff
@@ -127,6 +127,12 @@ services:
     environment:
       - CORS=true
 
+  mail:
+    image: mailhog/mailhog
+    container_name: 'ecamp3-mail'
+    ports:
+      - '8025:8025' # web UI
```

Comment:
@carlobeltrame would it be possible to include this in the dev deployment and expose this port, for example as `mail-dev.ecamp3.ch`?

Reply:
Können wir hier auch einen 3000-er Port verwenden?
So dass alle eCamp3-Dienste unter einem 300x Port laufen. Macht es etwas einfacher, sich die Ports zu merken.

Reply:
Klar, wir können lokal mappen was uns beliebt. Oder geht es dir auch um den Port im Container?

Reply:
#1419